src/views/SpaceCreate.vue

```diff
@@ -4,6 +4,8 @@ import { getInstance } from '@snapshot-labs/lock/plugins/vue3';
 import { PROPOSAL_QUERY } from '@/helpers/queries';
 import { proposalValidation } from '@/helpers/snapshot';
 import { ExtendedSpace } from '@/helpers/interfaces';
+import Plugin from '@/plugins/safeSnap';
+const safeSnapPlugin = new Plugin();
```

Comment:
Imports should be separated from others code
```suggestion
import Plugin from '@/plugins/safeSnap';

const safeSnapPlugin = new Plugin();
```